ygnmi/ygnmi.go

```diff
@@ -570,3 +571,51 @@ func BatchDelete[T any](sb *SetBatch, q ConfigQuery[T]) {
 		mode: deletePath,
 	})
 }
+
+// Batch contains a collection of paths.
+// Calling State() or Config() on the batch returns a query
+// that can be used to Lookup, Watch, etc on multiple paths at once.
+type Batch[T ygot.ValidatedGoStruct] struct {
```

Comment:
I see there's no attempt here to unify this or share code with the existing generated Batch type. It took me a bit to wrap my mind around why that has to be the case. I suppose the only way to get greater consistency or reuse the types would be to either (1) generate separate root State and Config batches instead of one; (2) make PathStruct generic and use that as the "root" of a Batch. Is that right? Neither seems super compelling.

Reply:
yeah, i couldn't see a way to do that was nice